bionemo-recipes/recipes/llama3/hydra_config/L0_convergence_ddp.yaml

```diff
@@ -0,0 +1,81 @@
+# @package _global_
+
+# Convergence test configuration for DDP with tiny Llama model (~10M params)
+# Tests that the model can overfit on a small 200MB dataset
+# Single GPU version
+
+defaults:
+  - defaults
+  - _self_
+
+# Use tiny Llama config for fast convergence testing
+model_tag: /workspaces/bionemo-framework/bionemo-recipes/recipes/llama3/tiny_llama_config
+
+# Training steps - enough to see convergence on small dataset
+num_train_steps: 1000
+
+# Dataset configuration - use 2MB subset
+dataset:
+  tokenizer_path: /workspaces/bionemo-framework/bionemo-recipes/models/llama3/nucleotide_fast_tokenizer
```

Comment:
same here, use a relative path. I think we want something similar to the `example_8m_checkpoint` directory we use in the esm2 examples